doc/nrf-bm/app_dev/dfu/memory_partitioning.rst

```diff
@@ -0,0 +1,82 @@
+.. _dfu_memory_partitioning:
+
+Memory Partitioning for DFU
+###########################
+
+In order to successfully run the DFU process, the memory of the device must be first correctly partitioned to accommodate all required components.
+
+|BMshort| uses Zephyr's `DTS`_ system for memory partitioning.
+You can find a practical example of how to partition the memory in chapter :ref:`ug_dfu_preparing_dfu_board`.
+
+Partitions overview
+*******************
+
+The following are detailed descriptions of each memory partition needed when running Firmware Over The Air (FOTA) updates with MCUboot as the Initial Root of Trust (IRoT).
+
+.. figure:: ../../images/partition_RRAM.svg
+   :alt: Memory partitioning with FOTA and MCUboot as IRoT
+
+The memory in the device is divided into several partitions, each serving a specific function in the DFU process:
+
+.. list-table::
+   :header-rows: 1
+
+   * - Partition name
+     - Purpose
+     - Details
+   * - boot_partition
+     - Reserved exclusively for IRoT.
+     - Starts at address 0x0000, size is determined by the needs of the MCUboot firmware.
+   * - slot0_partition
+     - Contains the main application firmware.
+     - Begins immediately after the storage partition or the IRoT partition if the storage is not enabled.
+       The size is maximized up to the start of the firmware loader.
+   * - slot1_partition
+     - Serves as the location for the DFU mode and contains the Firmware Loader.
+     - This partition contains the Firmware Loader that is validated by the IRoT.
+       The Firmware Loader is started when entering the DFU mode, while the SoftDevice is used by both the Firmware Loader and the Application that is running from ``slot0``.
```

Comment:
move softdevice description to softdevice partition

Reply:
done